ggshield/verticals/secret/secret_scanner.py

```diff
@@ -150,6 +151,8 @@ def _start_scans(
         chunks_for_futures = {}
 
         chunk: List[Scannable] = []
+        max_payload_size = self.client.maximum_payload_size
+        utf8_encoded_chunk_size = _SIZE_METADATA_OVERHEAD
```

Comment:
Nit: if you set `max_payload_size` to `self.client.maximum_payload_size - _SIZE_METADATA_OVERHEAD`, then `utf8_encoded_chunk_size` can be 0. The nice thing about this is `utf8_encoded_chunk_size` is really the sum of all document chunk sizes.

Reply:
very good point thanks !